pom.xml

```diff
@@ -789,6 +790,51 @@
                     <artifactId>build-helper-maven-plugin</artifactId>
                     <version>${maven-helper-plugin.version}</version>
                 </plugin>
+
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>license-maven-plugin</artifactId>
+                    <version>${maven-license-maven-plugin}</version>
+                    <executions>
+                        <execution>
+                            <id>download</id>
+                            <goals>
+                                <goal>download-licenses</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                    <configuration>
+                        <outputDirectory>${project.basedir}/seatunnel-dist/target/</outputDirectory>
+                        <thirdPartyFilename>THIRD-PARTY.txt</thirdPartyFilename>
+                        <sortArtifactByName>true</sortArtifactByName>
+                        <useMissingFile>false</useMissingFile>
+                        <addJavaLicenseAfterPackage>true</addJavaLicenseAfterPackage>
+                    </configuration>
+                </plugin>
+
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>exec-maven-plugin</artifactId>
+                    <version>${exec-maven-plugin.version}</version>
+                    <executions>
+                        <execution>
+                            <id>license</id>
+                            <phase>package</phase>
+                            <goals>
+                                <goal>exec</goal>
+                            </goals>
+                            <configuration>
```

Comment:
I suggest not to execute this goal by default, for some reasons:

- end users don't need to run this python script if they only want to download the source codes and build the package themselves
- the Python script will fail if the users don't have a correct Python version and dependency modules installed, the script is not vanilla Python script right?

Reply:
> end users don't need to run this python script if they only want to download the source codes and build the package themselves

Agree, I just disable these by default

Reply:
> the Python script will fail if the users don't have a correct Python version and dependency modules installed, the script is not vanilla Python script right?

About this, I believe there are two solutions:
- In future, we can use VirtualEnv to ensure the env is the same
- Use requirements file to help users quickly setup

Reply:
@CalvinKirs here, these comments you may want catch up.

Reply:
These are two different things, as I said in my email, it's just a CI check item. It is not executed by default, but you need to check it when running Github CI.

Reply:
@CalvinKirs Sure, the latest version has supported the Github Action to auto check, PTAL

Reply:
end users don't need to run this python script if they only want to download the source codes and build the package themselves
I think this config will still run the python script by default, am I right?

Reply:
@zhongjiajie @kezhenxu94 Since we are using Github Action to run the Python script instead of exec maven plugin, which is not necessary anymore, so I have removed the new exec maven plugin. PTAL the latest commit, thanks.